src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/LibraryResourceMenu.svelte

```diff
@@ -227,6 +245,31 @@
             isLoading = false;
         }
     }
+
+    function openResourceSearchMenu() {
+        setIsResourceSearch(true);
+    }
+
+    async function searchByParentResource(parentResource: ApiParentResource, offset: number | null) {
+        isLoading = true;
+        const currentLanguageId = get(currentLanguageInfo)?.id;
+        let response = (await fetchFromCacheOrApi(
+            ...searchResourcesEndpoint(
+                currentLanguageId,
+                '',
+                [parentResource.resourceType],
```

Comment:
Commented on the API PR too but instead of passing the resource types here, I think we'll want to pass the `parentResource.id` as a `parentResourceId` param.